tests/unit/helpers/questionnaires.test.js

```diff
@@ -335,3 +336,89 @@ describe('getUserQuestionnaires', () => {
     sinon.assert.notCalled(findOne);
   });
 });
+
+describe('getFollowUp', () => {
+  let courseFindOne;
+  let questionnaireFindOne;
+  beforeEach(() => {
+    courseFindOne = sinon.stub(Course, 'findOne');
+    questionnaireFindOne = sinon.stub(Questionnaire, 'findOne');
+  });
+  afterEach(() => {
+    courseFindOne.restore();
+    questionnaireFindOne.restore();
+  });
+
+  it('should return follow up', async () => {
```

Comment:
dans tes cas de tests tu peux ajouter 
- questionnaire.histories est une liste vide
- history.questionnaireAnswersList est une liste vide
- tu n'as pas traité ce cas la :    `if (answerList.length === 1 && !answerList[0].trim()) continue;`
- pour bien tester ce `if (!followUp[answer.card._id])` il faudrait mettre 2 reponses pour ce cas de test `it('should return follow up'` dans `questionnaireAnswersList`

Reply:
a la relecture, pour bien tester tes boucles for et le `if (!followUp[answer.card._id])` il faudrait une deuxieme historique de reponse

Reply:
done